implementations/python/mzlib/analyte.py

```diff
@@ -1,20 +1,194 @@
+import warnings
+
+try:
+    from collections.abc import (MutableMapping, Mapping)
+except ImportError:
+    from collections import (MutableMapping, Mapping)
+
 import textwrap
+from typing import Iterable, KeysView, ItemsView, ValuesView, Dict
+
+from mzlib.attributes import AttributedEntity, AttributeManager, IdentifiedAttributeManager
+
+
+FIRST_ANALYTE_KEY = '1'
```

Comment:
These identifiers are explicitly defined as integers in the specification. What is the advantage of encoding them as strings here?